README.md

```diff
@@ -50,47 +50,23 @@ NB: multiple CouchDB nodes will be more complicated, but the general pattern out
 
 By default CouchDB runs in "admin party" mode, which means you do not need users to read or edit any data. This is great for some, but to use Medic safely we're going to disable this feature.
 
-First, add an admin user. When prompted to create an admin during installation, use username:`admin` and password:`pass`. Passwords can be changed via [Fauxton](http://localhost:5984/_utils). For more information see the [CouchDB install doc](http://docs.couchdb.org/en/2.0.0/install/).
+First, add an admin user. When prompted to create an admin during installation, use a strong username and password. Passwords can be changed via [Fauxton](http://localhost:5984/_utils). For more information see the [CouchDB install doc](http://docs.couchdb.org/en/2.0.0/install/). 
 
-Now that's done, we must reconfigure CouchDB to require authentication:
+Now that's done, we must setup CouchDB:
```

Comment:
I didn't want to use the word setup here because CouchDB has its own setup that has to be run first and I didn't want to conflict with that.